### DISCUSSION TITLE
Cannot lock current layer

### DISCUSSION THREAD

**Discussion by gogamid** [manna-harbour/miryoku#334] [Q&A] [answered]
"The bottom row index and middle column keys are used for layer lock. Index is for the current layer. Middle is for the opposite layer (normally activated with the corresponding primary, secondary, or tertiary thumb key on the opposite hand) and can be used for single handed lock and layer use." 

I am able to lock the opposite layer, but locking the current layer doesn't do anything. How I do it:
I have mouse layer on the left side which is activated by holding right primary thumb key. If I want to lock mouse layer, I am still holding the right side primary thumb key and then tapping twice bottom row index. But this doesn't lock the mouse layer.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku_zmk#no-current-layer-lock.

**Comment by CupricReki** [comment]
Someone proposed a macro to solve this. I haven't tested this myself
https://github.com/zmkfirmware/zmk/issues/1299#issuecomment-2449934238

**Reply by manna-harbour** [reply]
There's also a PR in the linked issue. I may add support for that.
